### PR TITLE
continued preprocessing together...

### PR DESCRIPTION
**Pull request by pcfisch**
…up to the point where no missings anymore. To dos are given at end of notebook and also in teh kanban project

### CODE REVIEW

File: kickstarter/preprocessing.ipynb [hunk 1]
```diff
@@ -122,12 +128,231 @@
...
+    "# do we have only one target label in the missing usd_type - i.e. \"live\"?\n",
+    "idx_na = df_kick['usd_type'].isna()\n",
+    "df_kick.loc[idx_na,'state'].unique()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Loose missings of usd_type. will still have enough data points, especially since we are also diminishing imbalance by this step this seems fine.<br>\n",
```
Comment (millnattily) [inline kickstarter/preprocessing.ipynb:281]: lose (verlieren). loose(locker)

**Review by millnattily — APPROVED** [review]
just a spelling mistake 
looks great.

File: kickstarter/preprocessing.ipynb (hunk 1 above)
Comment (millnattily) [reply kickstarter/preprocessing.ipynb:281]: So if i've understood correctly you mean "get rid of" missings in usd_type column?  If that's the case then 'lose'.

Comment (pcfisch) [reply kickstarter/preprocessing.ipynb]: changed to lose

**Pull request merged**